src/products/iOS_products/review_iOSstrings.md

```diff
@@ -1,70 +1,74 @@
 # Reviewing strings for a new release of Mozilla iOS products
 
-Usually, a new release of our iOS products means an update to strings. If this is the case, a pull request will be made by the iOS team, in order to land these new strings, either in the [firefoxios-l10n](https://github.com/mozilla-l10n/firefoxios-l10n) or the [focusios-l10n](https://github.com/mozilla-l10n/focusios-l10n) repositories in `mozilla-l10n`.
+Usually, a new release of our iOS products means an update to strings. Like for Android products, mobile developers will land strings in their respective code repositories throughout the release cycle, and pull requests will be opened by GitHub automation in the iOS repositories to expose these strings for localization in Pontoon: either under the [firefoxios-l10n](https://github.com/mozilla-l10n/firefoxios-l10n) or the [focusios-l10n](https://github.com/mozilla-l10n/focusios-l10n) repositories in `mozilla-l10n`. Automation will try to import strings twice a week (Tuesdays and Thursdays), but it's also possible to manually trigger a string import to the `mozilla-l10n` repository any time by going to the GitHub `Actions` section of the iOS project and then importing strings again by running the string import workflow. It is always a good idea to import strings right before you review a PR, in case changes have been made in the meantime in the source repository. Example for Firefox for iOS [here](https://github.com/mozilla-l10n/firefoxios-l10n/actions/workflows/import_strings.yml).
```

Comment:
```suggestion
Usually, a new release of our iOS products means an update to strings. Like for Android products, mobile developers will land strings in their respective code repositories throughout the release cycle, and pull requests will be opened by GitHub automation in the iOS repositories to expose these strings for localization in Pontoon (either under the [firefoxios-l10n](https://github.com/mozilla-l10n/firefoxios-l10n) or the [focusios-l10n](https://github.com/mozilla-l10n/focusios-l10n) repositories in `mozilla-l10n`). Automation will try to import strings twice a week (Tuesdays and Thursdays), but it's also possible to manually trigger a string import to the `mozilla-l10n` repository any time by going to the GitHub `Actions` section of the l10n repository and then importing strings again by running the string import workflow. It is always a good idea to import strings right before you review a PR, in case changes have been made in the meantime in the source repository. Example for Firefox for iOS [here](https://github.com/mozilla-l10n/firefoxios-l10n/actions/workflows/import_strings.yml).
```

Out of curiosity, why do you think it's a good idea to run the import right before? If there are errors in the new strings imported, you're then stuck until they're fixed.

Reply:
In case new strings landed in the meantime, or there were changes to existing strings (happened in the past).
Didn't think about errors and being stuck, makes sense